pytest_plugins/jira_comments.py

```diff
@@ -81,21 +82,59 @@ def pytest_runtest_makereport(item, call):
             update_issue_to_tests_map(item, blocked_by_issues, test_result)
 
 
+# Function to escape special characters
+def escape_special_characters(text):
+    special_chars = ['[', ']']
```

Comment:
Minor, but if these are always going to be individual characters, then you can just use a simple string to iterate over.
```suggestion
    special_chars = '[]'
```

Reply:
Updated the PR